.stylelintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  // See https://github.com/wagtail/stylelint-config-wagtail for rules.
```

Comment:
I don't think this comment is necessary, most people reviewing this file would understand how the styelint extends system works.